doc/nrf/gsg_guides.rst

```diff
@@ -14,7 +14,7 @@ They provide a guided experience using Nordic Semiconductor tools and precompile
 This is not a comprehensive section that includes all of the available getting started guides for Nordic Semiconductor devices.
 
 .. note::
-   * To get started with the nRF9161 DK, complete the steps in the Quick Start app, available from `nRF Connect for Desktop`_.
+   * To get started with the nRF91x1 DKs, complete the steps in the `Quick Start`_ app, available from `nRF Connect for Desktop`_.
```

Comment:
We have nRF9131 as well... But it's an EK and not a DK though, so for now, probably ok :)

Reply:
Presently, only nRF9151, nRF9161 and nRF9160 have support for the Quick start app. I had checked with @martiles about the  GSG for the nRF9131, and he confirmed that since nRF9131 will also have Quick start support in the future, we don't need an GSG. We could update this page once the support is added.